homeassistant/components/roomba/__init__.py

```diff
@@ -40,15 +40,18 @@ def _has_all_unique_bilds(value):
     return value
 
 
-DEVICE_SCHEMA = vol.Schema(
-    {
-        vol.Required(CONF_HOST): str,
-        vol.Required(CONF_BLID): str,
-        vol.Required(CONF_PASSWORD): str,
-        vol.Optional(CONF_CERT, default=DEFAULT_CERT): str,
-        vol.Optional(CONF_CONTINUOUS, default=DEFAULT_CONTINUOUS): bool,
-        vol.Optional(CONF_DELAY, default=DEFAULT_DELAY): int,
-    },
+DEVICE_SCHEMA = vol.All(
+    cv.deprecated(CONF_CERT, invalidation_version="0.113"),
```

Comment:
I think the trend is to only set an `invalidation_version` if keeping it around actually causes problems: https://github.com/home-assistant/core/pull/33646#pullrequestreview-395010808

Reply:
That makes sense. Removing the `invalidation_version`.